mongo/integration/database_test.go

```diff
@@ -237,18 +237,32 @@ func TestDatabase(t *testing.T) {
 			assert.Nil(mt, err, "expected command %s to contain key 'batchSize'", evt.Command)
 		})
 
-		// The BatchSize option is not honored for ListCollections operations on server version 2.6 due to an
-		// inconsistency in the legacy OP_QUERY code path (GODRIVER-1937).
-		cmdMonitoringMtOpts := mtest.NewOptions().MinServerVersion("3.0")
+		cmdMonitoringMtOpts := mtest.NewOptions().MinServerVersion("2.6")
 		mt.RunOpts("getMore commands are monitored", cmdMonitoringMtOpts, func(mt *mtest.T) {
 			createCollections(mt, 2)
-			assertGetMoreCommandsAreMonitored(mt, "listCollections", func() (*mongo.Cursor, error) {
+
```

Comment:
This can move up outside the run blocks, so that you only need to do it once

Reply:
Maybe move it to a helper function if you also plan on running the previous "batch size" test for 2.6 so it can be shared across all of the tests? Unfortunately the best function name I can think of is `getListCollectionsCommandNameForMonitoring` which is very verbose, so definitely open to suggestions if you have them.

Reply:
I originally thought that the server version needed to be evaluated in the `Run` block, but it looks like it's a package-level call on `mtest`, not part of an `mtest.T` instance. Moved that server-version-specific command name check to before the run blocks.